Task2.py

```diff
@@ -0,0 +1,9 @@
+
+# Найдите сумму цифр трехзначного числа.
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+number = int(input('Введите трехзначное число: '))
+
+summ = int(number / 100) + number % 10 + number % 100 / 10
```

Comment:
Вы можете использовать оператор целочисленного отделения
```suggestion
summ = number // 100 + number % 10 + number % 100 // 10
```
Кроме того, вторую цифру можно найти по другому.
```suggestion
summ = number // 100 + number % 10 + number // 10 % 10
```